lib/perl/Genome/Model/Tools/Joinx/VcfMerge.pm

```diff
@@ -168,6 +173,9 @@ sub _resolve_flags {
     if ($self->merge_samples) {
         $flags .= " -s";
     }
+    if ($self->allow_same_file) {
```

Comment:
According to the [docs here](https://github.com/genome/genome/pull/672/files#diff-cf316049f5f32c0727e11cb88580cdbaR5), this feature flag requires joinx >= 1.11.

I see that immediately below this you're checking joinx version compatibility with a different flag (exact-pos). Should we be doing the same thing here?